packages/dashboard/components/consent/components/authForm/ParameterUpdateModal.tsx

```diff
@@ -1,13 +1,27 @@
 import React from 'react';
+import { useVersionEnabledCheck } from '../../hooks/useVersionEnabledCheck';
 
 interface ParameterUpdateModalProps {
   isOpen: boolean;
   onContinue: () => void;
   onUpdate: () => void;
   appName: string;
+  permittedVersion: number;
 }
 
-const ParameterUpdateModal = ({ isOpen, onContinue, onUpdate, appName }: ParameterUpdateModalProps) => {
+const ParameterUpdateModal = ({ 
+  isOpen, 
+  onContinue, 
+  onUpdate, 
+  appName, 
+  permittedVersion 
+}: ParameterUpdateModalProps) => {
+  const { isVersionEnabled } = useVersionEnabledCheck({
+    versionNumber: permittedVersion
+  });
+  
+  const isDisabled = isVersionEnabled === false;
```

Comment:
nit really:

```suggestion
  const isDisabled = !isVersionEnabled;
```
Or better just use it directly with the `!` below to avoid making a new variable

Reply:
Fixed!